src/main/java/com/microsoft/sqlserver/jdbc/IOBuffer.java

```diff
@@ -2234,48 +2337,58 @@ final void setNetworkTimeout(int timeout) throws IOException {
     }
 
-    // Thread pool - the values in the constructor are chosen based on the
-    // explanation given in design_connection_director_multisubnet.doc
+    /*
+     * Thread pool - the values in the constructor are chosen based on the explanation given in
+     * design_connection_director_multisubnet.doc
+     */
     private static final ThreadPoolExecutor threadPoolExecutor = new ThreadPoolExecutor(0, Integer.MAX_VALUE, 5,
             TimeUnit.SECONDS, new SynchronousQueue<Runnable>());
 
     // When parallel connections are to be used, use minimum timeout slice of 1500 milliseconds.
-    private static final int minTimeoutForParallelConnections = 1500;
+    private static final int MINTIMEOUTFORPARALLELCONNECTIONS = 1500;
 
-    // lock used for synchronization while updating
-    // data within a socketFinder object
+    /*
+     * lock used for synchronization while updating data within a socketFinder object
+     */
     private final Object socketFinderlock = new Object();
 
-    // lock on which the parent thread would wait
-    // after spawning threads.
+    /*
+     * lock on which the parent thread would wait after spawning threads.
+     */
     private final Object parentThreadLock = new Object();
 
-    // indicates whether the socketFinder has succeeded or failed
-    // in finding a socket or is still trying to find a socket
+    /*
+     * indicates whether the socketFinder has succeeded or failed in finding a socket or is still trying to find a
+     * socket
+     */
     private volatile Result result = Result.UNKNOWN;
 
-    // total no of socket connector threads
-    // spawned by a socketFinder object
+    /*
+     * total no of socket connector threads spawned by a socketFinder object
+     */
     private int noOfSpawnedThreads = 0;
 
-    // no of threads that finished their socket connection
-    // attempts and notified socketFinder about their result
+    /*
+     * no of threads that finished their socket connection attempts and notified socketFinder about their result
+     */
     private int noOfThreadsThatNotified = 0;
 
-    // If a valid connected socket is found, this value would be non-null,
-    // else this would be null
+    /*
+     * If a valid connected socket is found, this value would be non-null, else this would be null
+     */
     private volatile Socket selectedSocket = null;
 
-    // This would be one of the exceptions returned by the
-    // socketConnector threads
+    /*
+     * This would be one of the exceptions returned by the socketConnector threads
+     */
     private volatile IOException selectedException = null;
 
     // Logging variables
     private static final Logger logger = Logger.getLogger("com.microsoft.sqlserver.jdbc.internals.SocketFinder");
     private final String traceID;
 
     // maximum number of IP Addresses supported
-    private static final int ipAddressLimit = 64;
+    private static final int IPADDRESSLIMIT = 64;
```

Comment:
if you change the variable name to all caps, you need to put underscores in them